hd/etc/menubar.txt

```diff
@@ -117,6 +117,9 @@ fr: Saisissez une succession de lettres de module avec son chiffre d’option da
               <div class="dropdown-divider"></div>
               <a class="dropdown-item" href="%prefix;m=MRG;i=%index;" title="[*merge] %first_name;.%occ; %surname;"><span class="fa fa-user-times fa-fw text-danger"></span>  [*merge::person/persons]0</a>
               <a class="dropdown-item" href="%prefix;%access;;i=%index;;m=DEL_IND;" title="[*delete] %self;"><span class="fa fa-trash fa-fw text-danger"></span>  [*delete::person/persons]0</a>
+              %if;(bvar.can_kill_ancestors="yes")
+                <a class="dropdown-item" href="%prefix;%access;;i=%index;;m=KILL_ANC;" title="[*delete/deleted]0 [ancestors] [of] %self;"><span class="fa fa-trash fa-fw text-danger"></span>  [*delete/deleted]0 [ancestors]</a>
```

Comment:
So if I'm not wrong `[*delete/deleted]0 [ancestors] [of] %self;` would be "delete ancestors of XXX". But you created a special message for that. Also, you have no guarantee that x of y is the same in every languages, that's why you have `a_of_b` (that you should know well now). As a side note, why the spaces `</span>  [*delete/deleted]0 [ancestors]</a>`